requirements.txt

```diff
@@ -60,3 +60,6 @@ unidecode
 
 requests==2.23.0,<3.0
 responses==0.10.6
+
+# For Delta Lake
+pyspark==3.0.1
```

Comment:
could we make it a range or even put it in extra requirement like https://github.com/amundsen-io/amundsendatabuilder/blob/master/setup.py#L54 ?

Reply:
ah ok I will move it to extras